src/main/openapi/daam.yaml

```diff
@@ -526,13 +526,20 @@ components:
         title:
           type: string
           title: Application title
+        description:
```

Comment:
@brunopacheco1  the list api of REMS do not invlude events so we can't really show last event in list view easily.

I would say we rather show other general fields like description and created At and let the user see list of events in the detailed view

Reply:
Sure, np problem.